pkgs/games/sgfutils/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, openssl
+, iconv
+, makeWrapper
+, imagemagick
+, makeFontsConf
+}:
+stdenv.mkDerivation
+{
+  pname = "sgfutils";
+  version = "unstable-2017-11-27";
+  buildInputs = [ openssl makeWrapper ] ++ lib.optionals stdenv.isDarwin [ iconv ];
```

Comment:
Odd choice to put `buildInputs` up here, and not closer to `buildPhase`.

Reply:
Remains unresolved.

Reply:
I understand your opinion and I disagree with it. As there is no contributing stipulation that governs where buildInputs is, I consider it resolved.

Reply:
I agree this is minor and frankly I would probably not request this myself to be honest. Anyway as a gesture of good will I would suggest we follow @mweinelt's suggestion as a gesture of a good will since he did most of the work for reviewing this PR.

@ggPeti Could you move `buildInputs` above the `buildPhase`. I think we ideally want to make everyone involved happy with the merge.

Reply:
Certainly. I believe there's something to say for the role of maintainers though. It's a stylistic difference that is very likely to only matter to the person maintaining the package (me). To unresolve my decision and assert dominance over this insubstantive matter is quite petty and unproductive. But I'm down to make people happy.